app/resources/api/v2/plate_purpose_resource.rb

```diff
@@ -13,34 +22,34 @@ class PlatePurposeResource < BaseResource
 
       # The following attributes are sent by Limber for a new plate purpose.
 
-      # @!attribute [rw]
-      # @return [String] The name of the plate purpose.
+      # @!attribute [rw] name
```

Comment:
is [rw] the default?

Reply:
I think it is, yeah.  However YARD does say you can put `[rw]` if you want, and I think it's good to have the access modifier listed for all of them rather than only when it's absolutely necessary.